continuous_integration/environment-mindeps-optional.yaml

```diff
@@ -24,24 +24,24 @@ dependencies:
   # optional dependencies used by dask
   - cachey=0.1.1
   - crick=0.0.3
-  - cytoolz=0.10.1
+  - cytoolz=0.11.0
   - dask-ml=1.4.0
-  - fastavro=0.22.6
+  - fastavro=1.1.0
```

Comment:
I'm curious about why these non-`python` version bumps are needed. The actual bumps here look fine, I'm just looking for some extra context

Reply:
these packages don't have py3.9 packages on conda-forge until these versions